av_copy_packet: not available on ubuntu 14.04

```
/home/fab/dev/TuttleOFX/plugins/image/io/AudioVideo/avTranscoder/src/AvTranscoder/frame/Frame.cpp: In constructor ‘avtranscoder::Frame::Frame(AVPacket&)’:
/home/fab/dev/TuttleOFX/plugins/image/io/AudioVideo/avTranscoder/src/AvTranscoder/frame/Frame.cpp:18:38: error: ‘av_copy_packet’ was not declared in this scope
  av_copy_packet( &_packet, &avPacket );
                                      ^
```

http://ffmpeg.org/pipermail/ffmpeg-cvslog/2012-September/054806.html


<!---
@huboard:{"milestone_order":80.0,"order":86}
-->

--- a/src/AvTranscoder/frame/Frame.cpp
+++ b/src/AvTranscoder/frame/Frame.cpp
@@ -15,7 +15,13 @@ Frame::Frame( const size_t dataSize )
 
 Frame::Frame(AVPacket& avPacket)
 {
+#if LIBAVCODEC_VERSION_MAJOR > 54 || ( LIBAVCODEC_VERSION_MAJOR == 54 && LIBAVCODEC_VERSION_MINOR > 56 )
 	av_copy_packet( &_packet, &avPacket );
+#else
+	// we just care about data, not side properties of AVPacket
+	initAVPacket();
+	copyData( avPacket.data, avPacket.size );
+#endif
 }
 
 Frame::~Frame()

--- a/src/AvTranscoder/frame/Frame.hpp
+++ b/src/AvTranscoder/frame/Frame.hpp
@@ -19,7 +19,7 @@ public:
 	/// Create a frame with a the given buffer size
 	Frame( const size_t dataSize );
 
-	/// Create a frame from the given AVPAcket (copy data)
+	/// Create a frame from the given AVPAcket (copy data of given packet)
 	Frame( AVPacket& avPacket );
 
 	/// Free buffer of data

